### DISCUSSION TITLE
GCC version

### DISCUSSION THREAD

**Discussion by rousseldenis** [GrandOrgue/grandorgue#1766] [Developers] [answered]
@oleg68 I don't remember the current of version of gcc we are using. Could you point me to it?

**Comment by oleg68** [comment, marked as answer]
Now we use c++ 11 standard. So we can use any gcc version that supports this standard https://gcc.gnu.org/projects/cxx-status.html

Github builds are performed eith gcc 9.4.

**Reply by rousseldenis** [reply]
Great, thanks
